db/schema.rb

```diff
@@ -44,47 +43,47 @@
     t.string "name"
     t.string "email"
     t.boolean "deleted", default: false
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at", precision: nil, null: false
+    t.datetime "updated_at", precision: nil, null: false
   end
 
   create_table "deputy_directors", id: :serial, force: :cascade do |t|
     t.string "name"
     t.string "email"
     t.integer "division_id"
     t.boolean "deleted", default: false
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at", precision: nil, null: false
+    t.datetime "updated_at", precision: nil, null: false
   end
 
   create_table "directorates", id: :serial, force: :cascade do |t|
     t.string "name"
     t.boolean "deleted", default: false
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at", precision: nil, null: false
+    t.datetime "updated_at", precision: nil, null: false
   end
 
   create_table "divisions", id: :serial, force: :cascade do |t|
     t.string "name"
     t.integer "directorate_id"
     t.boolean "deleted", default: false
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at", precision: nil, null: false
+    t.datetime "updated_at", precision: nil, null: false
   end
 
   create_table "early_bird_members", id: :serial, force: :cascade do |t|
     t.string "name"
     t.string "email"
     t.boolean "deleted"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at", precision: nil, null: false
+    t.datetime "updated_at", precision: nil, null: false
   end
 
   create_table "early_bird_organisers", force: :cascade do |t|
     t.date "date_from"
     t.date "date_to"
-    t.datetime "created_at", precision: 6, null: false
-    t.datetime "updated_at", precision: 6, null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
```

Comment:
Should this be precision: nil?

Reply:
There was a change in Rails 7 so that the precision value is set to 6 by default. Running `rails db:migrate` automatically updates schema.rb to reflect that which is why it has changed. The actual value in the database will not change.